AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/ReconcileAndLocalSave/AWSModelReconciliationQueue.swift

```diff
@@ -141,6 +143,22 @@ final class AWSModelReconciliationQueue: ModelReconciliationQueue {
             modelReconciliationQueueSubject.send(completion: .failure(dataStoreError))
         }
     }
+
+    func kill() {
+        incomingEventsSink?.cancel()
+        incomingEventsSink = nil
+
+        incomingSubscriptionEvents.kill()
+
+        //Enqueue the rest of the items that our subscription downloaded
```

Comment:
I'm not sure this is the right behavior. The data contained in these subscriptions are going to be re-populated on the next startup by the delta/base query, and it's probably more important to exit quickly than wait for incoming mutations to apply.

Reply:
Ah, good point, our initial sync operation should handle that -- I'll change these to cancel all operations.